ext/date/tests/date_debugInfo.phpt

```diff
@@ -0,0 +1,59 @@
+--TEST--
+Bug GH-11310: __debugInfo() does nothing on classes extending DateTime
+--INI--
+date.timezone=UTC
+--FILE--
+<?php
+
+class MyDateTime extends DateTime {
+    public function __debugInfo(): array {
+        return ['child' => '42', 'parent' => count(parent::__debugInfo())];
+    }
+}
+
+class MyDateTimeImmutable extends DateTimeImmutable {
+    public function __debugInfo(): array {
+        return ['child' => '42', 'parent' => count(parent::__debugInfo())];
+    }
+}
+
+class MyDateTimeZone extends DateTimeZone {
```

Comment:
suggest adding a test case for MyDateTimeZone, otherwise why define the class at all?